7-CSS-Box-Model/Part-2/css/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  margin: 0;
+  padding: 0;
+  transition: all 0.1s ease-in;
+}
+
+section {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  font-size: 0%;
+  box-sizing: border-box;
+  border: solid 15px #2b81ac;
+  background-color: #3399cc;
+  color: white;
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+}
+
+span {
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+}
+
+h1 {
+  font-size: 25px;
+  margin: 2% 20%;
+}
+
+h1 > i {
+  margin-left: 10px;
+}
+
+ul li {
+  display: inline-block;
+}
+
+ul {
+  position: fixed;
+  left: 20%;
+  width: 60%;
+  height: 70%;

```

Comment:
it's incorrect way to handle it, you should rely on content inside of li, not apply height for ul, let's keep it and discuss on the lesson